src/magfie/bdivfree.f90

```diff
@@ -181,6 +181,51 @@ subroutine vector_potentials(nr_in,np_in,nz_in,ntor_in,      &
 102 format(1000e15.7)
 end subroutine vector_potentials
 
+
+! Spline a single toroidal Fourier mode of the RMP coil field, as needed for MEPHIT.
+! Note the explicit-shape input arrays are not checked for shape.
+! Also, the truncation of data outside the limiting convex is not performed.
+subroutine vector_potential_single_mode(ntor_in, nR_in, nZ_in, Rmin_in, Rmax_in, Zmin_in, Zmax_in, Bn_R, Bn_Z)
```

Comment:
If you spline this, why not call the routine `spline_vector_potential_single_mode`?

Reply:
For consistency with the other routines in this file. It's a single-mode analogue of `vector_potentials`, while the `spline_*` routines are the ones to evaluate the splines. I don't know who else uses these functions.